llmobs/internal/errors/errors.go

```diff
@@ -0,0 +1,75 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025 Datadog, Inc.
+
+package errors
```

Comment:
We have a [similar package](https://github.com/DataDog/dd-trace-go/pull/3709) that we merged a bit ago. Could it cover the use cases we have here, instead of creating a new package for it? We can add more functionality to the other package as needed.

Reply:
yes! I didn't know this existed. I already refactored the code to use this package, will push the changes shortly 🙏